### DISCUSSION TITLE
feat: Location Button for Simple Location Information

### DISCUSSION THREAD

**Discussion by brndnz857** [acikkaynak/deprem-yardim-frontend#720] [Ideas]
**discord username: HayırSever#1421** 

## Define your request

It is aimed to get the location information directly from google maps with a button. It can be thought of as sending location information via Whatsapp.

## Other Evaluated Alternatives
Alternatively, Yandex maps can be added. In terms of integration, whichever api solution will be faster can be preferred.

## Extra context
A reference from discord by @eraygundogmus:
https://discord.com/channels/1072074800622739476/1072590684223184989/1073664787310981142

**Comment by 0xCakin** [comment]
Moving this item to ideas.

**Comment by 0xCakin** [comment]
Released.
